legacy_ibc_testing/testing/events.go

```diff
@@ -69,7 +69,7 @@ func ParseAckFromEvents(events sdk.Events) ([]byte, error) {
 	for _, ev := range events {
 		if ev.Type == channeltypes.EventTypeWriteAck {
 			for _, attr := range ev.Attributes {
-				if string(attr.Key) == channeltypes.AttributeKeyAck {
+				if attr.Key == channeltypes.AttributeKeyAckHex {
```

Comment:
Same question as above. Maybe the two changes are tied together but I'm evaluating this as a linting / fully refactor PR